File: Java/T5/T05_Dialogos/app/src/main/java/com/borja/dialogos/dialogos/DialogoInformacion.java
package com.borja.dialogos.dialogos;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.Context;
import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.DialogFragment;

import com.borja.dialogos.R;

public class DialogoInformacion extends DialogFragment {



    @NonNull
    @Override
    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {

        AlertDialog.Builder dialogo = new AlertDialog.Builder(getContext());

        dialogo.setTitle(R.string.titulo_dialogo_info);
        dialogo.setMessage(R.string.mensaje_dialogo_info);
        //dialogo.setMessage(getResources().getString(R.string.mensaje_dialogo_info));
        return dialogo.create();
    }
}

File: Kotlin/T3/codigos_ces/Tienda/app/src/main/java/com/example/tienda/ui/dialogs/DialogoInformacion.kt
package com.example.tienda.ui.dialogs

import android.app.Dialog
import android.os.Bundle
import android.util.Log
import android.view.View
import androidx.appcompat.app.AlertDialog
import androidx.fragment.app.DialogFragment

class DialogoInformacion : DialogFragment() {

    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {

        val buider: AlertDialog.Builder = AlertDialog.Builder(requireContext())

        buider.setTitle("Informacion de la APP")
        buider.setMessage("Esta app ha sido realizada para analizar datos de recycler view")
        buider.setPositiveButton("OK", { _, _ -> {
            Log.v("dialogo","Pulsado el boton positivo")
        } })
        buider.setNegativeButton("NO", { _, _ -> {
            Log.v("dialogo","Pulsado el boton negativo")
        } })

        buider.setNeutralButton("CANCEL", { _, _ -> {
            Log.v("dialogo","Pulsado el boton cancelar")
        } })

        return buider.create()
    }

}